### DISCUSSION TITLE
API documentation

### DISCUSSION THREAD

**Discussion by jotepen** [wgu-opensource/osmt#117] [Asking for Help] [answered]
I wanted to see if there is any documentation for making API calls to my OSMT server.  In the README file for the API section, I see that it allows for anonymous API connections (Allowing anonymous API to search and list endpoints for published skills and collections), but nothing on how to actually make the API calls.  Is this feature available yet or did I miss the documentation?

Thank you!

**Comment by JohnKallies** [comment, marked as answer]
Hi Joshua.

Please see https://github.com/wgu-opensource/osmt/issues/110. There's an open issue for this question. There are some OpenAPI yml files in the ./docs directory, but I can't speak to how complete or accurate they are. Ideally, we would have something in the Maven build that validates against them.

We will move forward from our "soft open" in the coming weeks, and then be able to allow forking and accept PRs. Please watch the issue above for traction around your question?
